pkgs/shells/nushell/plugins/plot.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+  runCommand,
+  nushell,
+  plot,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "nu_plugin_plot";
+  version = "0-unstable-2025-06-11";
```

```suggestion
  version = "0.103.0-unstable-2025-06-11";
```
You can use a version from the manifest (but it was set earlier, hence stays unstable suffix)